components/eamxx/tests/uncoupled/mam4/mam4_nucleation_standalone.cpp

```diff
@@ -38,10 +39,11 @@ TEST_CASE("mam4-nucleation-standalone", "") {
   const auto  t0_str = ts.get<std::string>("run_t0");
   const auto  t0     = util::str_to_time_stamp(t0_str);
 
+  logger.info("running MAMMicrophysics standalone test with dt = {} for {} steps.", dt, nsteps);
+
   // Need to register products in the factory *before* we create any atm process or grids manager.
-  auto& proc_factory = AtmosphereProcessFactory::instance();
+  register_physics();
   auto& gm_factory = GridsManagerFactory::instance();
```

Comment:
To mimic the line above, where you used `register_physics` to hide tedious lines that the end user should not care about, you could simply do
```
register_mesh_free_grids_manager();
```
rather than do it manually.